src/index.html

```diff
@@ -3,12 +3,80 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#">
+        <img src="./images/logo.png"
+        class="logo-img"
+        alt="logo"
+        >
+      </a>
```

Comment:
👀 
```suggestion
      <a href="#">
        <img 
          src="./images/logo.png"
          class="logo-img"
          alt="logo"
        >
      </a>
```

Reply:
also, you should be more specific in alt attribute. It is alternate description for the image - so I should understand exactly what that is without seeing the picture
```suggestion
      <a
        href="#">
        <img src="./images/logo.png"
        class="logo-img"
        alt="MOYO logo"
        >
      </a>
```